arkouda/pdarrayclass.py

```diff
@@ -399,6 +395,18 @@ def max_bits(self, max_bits):
             generic_msg(cmd="set_max_bits", args={"array": self, "max_bits": max_bits})
             self._max_bits = max_bits
 
+    def equals(self, other) -> bool:
+        """
+        Whether pdarrays are the same size and all entries are equal.
+        """
+        if isinstance(other, pdarray):
+            if other.size != self.size:
+                return False
+            else:
+                return all(self == other)
+        else:
+            return False
```

Comment:
it seems code from https://github.com/Bears-R-Us/arkouda/pull/3209 has slipped into this PR

Reply:
Sorry, I guess I was rushing this one a bit.  I did remove it.

Reply:
no worries!!